README.md

```diff
@@ -1,19 +1,17 @@
-# fragment
-Python Flask project used to aggregate and distribute IRC meeting minutes and logs for the Fedora Project
-
-## How to set up?
+# Mote 2.0
 
-1. `git clone https://github.com/t0xic0der/fragment.git`
-2. `cd fragment/`
-3. `virtualenv venv`
-4. `source venv/bin/activate`
-5. `pip3 install -r requirements.txt`
-6. `python3 main.py --help`
-7. `python3 main.py -p 9696 -4`
+Python Flask project used to aggregate and distribute IRC meeting minutes and logs for the Fedora Project
 
-## How to test changes locally using Podman?
+## How to set up a local development environment?
 
-1. Build the image `podman build . -f Dockerfile -t fedora-easyfix:latest`.
-2. Start the server `podman run -p 9696:9696 -ti fedora-easyfix:latest -p 9696 -4`
+1. Download https://drive.google.com/file/d/1iOKEgyHMtpnCwsuZa5ufT8gI9-ntuOVc/view?usp=sharing.
```

Comment:
You can't access this link without Google account

Reply:
I have now moved the archive to Mega. Please let me know if any more changes are needed.